sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/ResolveDefaultColumns.scala

```diff
@@ -153,8 +153,10 @@ case class ResolveDefaultColumns(catalog: SessionCatalog) extends Rule[LogicalPl
     val schemaForTargetTable: Option[StructType] = getSchemaForTargetTable(u.table)
     schemaForTargetTable.map { schema =>
       val defaultExpressions: Seq[Expression] = schema.fields.map {
-        case f if f.metadata.contains(CURRENT_DEFAULT_COLUMN_METADATA_KEY) => analyze(f, "UPDATE")
-        case _ => Literal(null)
+        case f if f.metadata.contains(CURRENT_DEFAULT_COLUMN_METADATA_KEY) =>
+          analyze(f, CURRENT_DEFAULT_COLUMN_METADATA_KEY, "UPDATE")
+        case _ =>
+          Literal(null)
```

Comment:
```suggestion
        case _ => Literal(null)
```

Reply:
@beliefer this is minor. Usually, it is up to the contributors.

Reply:
No worries, done. Any code review comment is welcome :) I appreciate the effort it takes to review the new code.

Reply:
:)